modules/ROOT/edits/limitations-self.adoc

```diff
@@ -0,0 +1,19 @@
+= Limitations with Anypoint Runtime Fabric on Self-Managed Kubernetes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following information is important when installing, configuring, and managing Anypoint Runtime Fabric on Self-Managed Kubernetes.
+
+[cols="2*a"]
+.Anypoint Runtime Fabric on Self-Managed Kubernetes resource allocation 
```

Comment:
GLOBAL: remove table and figure titles in favor of just introducing them in the narrative.